backend/benefit/messages/serializers.py

```diff
@@ -0,0 +1,72 @@
+from applications.enums import ApplicationStatus
+from applications.models import Application
+from django.utils.translation import gettext_lazy as _
+from messages.models import Message, MessageType
+from rest_framework import serializers
+from rest_framework.exceptions import NotFound, PermissionDenied
+from users.utils import get_company_from_request, get_request_user_from_context
+
+
+class MessageSerializer(serializers.ModelSerializer):
+    message_type = serializers.ChoiceField(
+        choices=MessageType.choices, help_text="Type of the message"
+    )
+
+    class Meta:
+        model = Message
+        fields = [
+            "id",
+            "created_at",
+            "modified_at",
+            "content",
+            "message_type",
```

Comment:
Is it so that also the handler UI doesn't need the "sender" field?

Reply:
At least from the design I don't see the place where sender is needed. However, the NoteSerializer has `sender`